### PR TITLE
Update User Guide (Comment Command)

### PR DESCRIPTION
**Pull request by aweijun**
## Details
* Update table of contents in UserGuide
* Update the Command Summary of UserGuide.
* Fixed example of edit in Command Summary.
* Add comment command under Features Component.
* Add comment command under Table of Content
* Modified Add Show to account for comment.
* Modified Edit Show to account for comment.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -193,6 +194,23 @@ Examples:
...
+Edit comment of a specific show from Trackermon.
+
+Format: `comment <INDEX> [c/<COMMENT>]`
+* Edit comment of the show at the specified `<INDEX>`.
+* The index refers to the index number shown in the displayed show list. (not overall)
+* The index **must be a positive integer** 1,2,3,...
+* Omitting the `[c/<COMMENT>]` would remove the comment of that specific show.
+
+Examples:
+* `list` followed by `comment 2 c/Shags` edit 2nd show's comment in Trackermon to "Shags".
```
Comment (Ardentsoul) [inline docs/UserGuide.md:209]: Should this be `edits the` instead of `edit`?

Comment (aweijun) [reply docs/UserGuide.md:209]: I will edit it ASAP.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -225,14 +243,15 @@ _Details coming soon ..._
...
-| **List**   | `list`                                                                                                                                                |
+| Action      | Format, Examples                                                                                                                                      |
+|-------------|-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**     | `add n/<NAME> s/<STATUS> [c/<COMMENT>] [t/<TAG>]…​` <br> e.g., `n/ReZero s/watching c/Good Show t/Anime`                                              |
+| **Delete**  | `delete <INDEX>`<br> e.g., `delete 3`                                                                                                                 |
+| **Edit**    | `edit <INDEX> [n/<NAME>] [s/<STATUS>] [c/<COMMENT>] [t/<TAG>]…​` <br> e.g., `edit 1 n/ReZero s/watching c/Good Show t/Anime`                          |
+| **Comment** | `comment <INDEX> [c/<COMMENT>]`  eg., `comment 2 c/Bad`                                                                                               |
```
[the remark this answers is not in the document]
Comment (aweijun) [reply docs/UserGuide.md:269]: This has been edited in the master branch already

File: docs/UserGuide.md [hunk 3]
```diff
@@ -17,6 +17,7 @@ They can easily look up the list of shows if they need to.
   * [Finding a show: `find`](#finding-a-show-find)
   * [Deleting a show: `delete`](#deleting-a-show-delete)
   * [Editing a show: `edit`](#editing-a-show-edit)
+  * [Editing a comment: `comment`](#editing-a-comment-comment)
```
Comment (JonathanHoshi) [inline docs/UserGuide.md:20]: Maybe you could change it to `Commenting a show:` instead?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -193,6 +194,23 @@ Examples:
 
 [return to top <img src="images/toc-icon.png" width="25px">](#table-of-contents)
 
+---
+### Editing a comment: `comment`
```
Comment (JonathanHoshi) [inline docs/UserGuide.md:198]: Maybe you could change it to `Commenting a show:` instead?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -251,6 +280,7 @@ _Details coming soon ..._
 | **Find**   | `find <KEYWORD>`<br> e.g., `find hero`<br><br>`find [n/NAME] [s/STATUS] [t/<TAG>]…​`<br>e.g., `find n/Shingeki no kyojin s/watching t/Anime t/Seinen` |
 | **List**   | `list`                                                                                                                                                |
 | **Sort**   | `sort [sna/] [snd/] [ssa/] [ssd/] [so/]` |
+>>>>>>> master
```
Comment (JonathanHoshi) [inline docs/UserGuide.md:283]: Did you forgot to resolve your merge conflicts?

**Review by JonathanHoshi — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 3 above)
Comment (aweijun) [reply docs/UserGuide.md:19]: Good point

**Review by JonathanHoshi — APPROVED** [review]
LGTM!

**Pull request merged**